theme/home.less

```
@@ -158,8 +161,10 @@
                     font-weight: 500;
                     border-bottom: 1px solid fade(@primaryColor, 0%);
                     transition: border-color 400ms ease;
-                    &:hover, &:focus {
+                    &:hover, &:focus-visible {
                         border-color: @primaryColor;
+                        border: 2px solid white;
+                        outline: @homeCardBorderSize solid @homeCardFocusBorderColor !important;
```

The new styling needs splitting from the existing hover & focus border style which is just underlining the text using a border for some reason. We can also substitute border for outline-offset again.
```suggestion
                    &:hover, &:focus {
                        border-color: @primaryColor;
                    }
                    &:focus-visible {
                        outline: @homeCardBorderSize solid @homeCardFocusBorderColor !important;
                        outline-offset: 2px;
```